panel/1.0/webserver_configuration.md

```diff
@@ -109,6 +113,50 @@ sudo a2enmod rewrite
 sudo systemctl restart apache2
 ```
 
+:::
+::: tab "Caddy With Automatic SSL"
+
+You should paste the contents of the file bellow, replacing `<domain>` with your domain name.
+
+The default config path is `/etc/caddy/Caddyfile`.
+
+<<< @/.snippets/webservers/Caddyfile{9}
+
+::: tip
+If you are using Cloudflare DNS in proxy mode, refer to [this tutorial](/tutorials/creating_ssl_certificates.html#method-3:-caddy-(using-cloudflare-api)), to see how to configure Caddy to use DNS challenge for obtaining SSL certificates.
+:::
+
+### Enabling Configuration
+
+The final step is to validate configuration and restart Caddy.
+
+```bash
+caddy validate --config /etc/caddy/Caddyfile
```

Comment:
Same here, shouldn't be the default file? Also as NGINX/Apache aren't being validated, why do it in Caddy?

Reply:
It is a good/best practise to validate your configuration before applying it to live environment. You can catch syntax errors, etc. before you deploy the changes.

I can send another PR to add validation for NGINX. And perhaps Apache, but I've not used Apache in very long time tho.

Reply:
Refactored: dropped config validation for simplicity.